File: Python/LargestOfTwoNumbers.py
# Challenge 12: Largest of Two Numbers

# Find the larger value between two numbers.

# Example:
# Input:
# 15
# 7

# Output:
# 15

number = [15, 7, 100]
max = number[0]
n = len(number)

for i in range(1, n):
  if number[i] > max:
    max = number[i]

print(max)

File: JavaScript/LargestOfTwoNumbers.js
/*
Challenge 12: Largest of Two Numbers

Find the larger value between two numbers.

Example:
Input:
15
7

Output:
15
*/

let number = [15, 20, 1]
let max = number[0]

for (let i = 1; i < number.length; i++) {
  if (number[i] > max) {
    max = number[i]
  }
}

console.log(max)